resources/backup_scripts/db-backup.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
```

Comment:
@yonas I'm not seeing where the database dump is generated. There is one DB in the bucket that's 7 bytes. We need to add some documentation in a README indicating how database dumps are generated, what the desired cron job should look like, etc.